signer-npm/js/src/utils.js

```diff
@@ -111,7 +111,7 @@ function bytesToAddress(payload, testnet) {
       }
       break;
     case ProtocolIndicator.ACTOR:
-      if (payload.slice(1).length !== 32) {
+      if (payload.slice(1).length !== 20) {
```

Comment:
Is it sure that actor protocol payload are 20 bytes ? https://filecoin-project.github.io/specs/#protocol-2-actor

The payload is a sha256 which if I not mistaken is 32 bytes ?

Reply:
Yes, this does not seem correct...
Maybe @mrsmkl can provide more info?

Reply:
In the go implementation the actor address as computed similarly to secp256k1 address, https://github.com/filecoin-project/go-address/blob/master/address.go#L166

Also in your test cases, for example https://github.com/Zondax/filecoin-signing-tools/blob/dev/examples/manual_testvectors.json#L142
... that's why I changed this. Let's ask about whether the specs should be updated.